### DISCUSSION TITLE
Shortcut to grab avatar from Mastodon

### DISCUSSION THREAD

**Discussion by jeyemwey** [Traewelling/traewelling#2015] [Ideas / Feature Requests]
**Is your feature request related to a problem? Please describe.**
I'd like to set my social avatar as my Träwelling image, but I don't have it handy.

**Describe the solution you'd like**
If my Träwelling account connected to my Mastodon Account, I'd like to have a shortcut that grabs my avatar and uploads it to Träwelling. It's squared already, so no more steps needed.

**Describe alternatives you've considered**
--

**Additional context**
* [Related Gitlab Issue](https://gitlab.com/HerrLevin_/trwl/issues/94)
* [Possible code change, needs revision](https://gitlab.com/HerrLevin_/trwl/-/merge_requests/160)

**Comment by HerrLevin** [comment]
This is currently on hold since it's not compliant with our GDPR notice.

**Reply by MrKrisKrisu** [reply]
In my opinion, that would be compliant, since the user explicitly clicks it themselves, which would be equivalent to them simply uploading it directly.

**Comment by MrKrisKrisu** [comment]
> This is currently on hold since it's not compliant with our GDPR notice.

Is there any update on this issue? Can we proceed with implementing this feature or do we need to take additional steps to comply with GDPR?
